cellcom/Cargo.toml

```diff
@@ -18,3 +23,5 @@ serialport = "4.6.1"
 tracing.workspace = true
 tracing-subscriber.workspace = true
 wpactrl = "0.5.1"
+orb-endpoints.workspace = true
+orb-security-utils = { workspace = true, features = [ "reqwest", "blocking" ] }
```

Comment:
I think all code we write should generally use async for stuff that does io. But I don't want to drag down the PR. So I will not block it.